pkg/kubelet/dockertools/docker.go

```diff
@@ -325,7 +326,7 @@ func NewDockerContainerCommandRunner(client DockerInterface) ContainerCommandRun
 }
 
 func (p dockerPuller) Pull(image string) error {
-	image, tag := parseImageName(image)
+	_, tag := parsers.ParseRepositoryTag(image)
```

Comment:
Somehow I thought there is a unittests for this instead of removing one. My bad not catch the issue.